src/vs/workbench/parts/preferences/browser/keybindingsEditor.ts

```diff
@@ -259,19 +259,49 @@ export class KeybindingsEditor extends BaseEditor implements IKeybindingsEditor
 		this.searchWidget.clear();
 	}
 
-	showSimilarKeybindings(keybindingEntry: IKeybindingItemEntry): TPromise<any> {
-		const value = `"${keybindingEntry.keybindingItem.keybinding.getAriaLabel()}"`;
+	showSimilarKeybindings(keybindingEntry: IKeybindingItemEntry | [ResolvedKeybinding, ResolvedKeybinding]): TPromise<any> {
+		let value = '';
+		if ('keybindingItem' in keybindingEntry) {
+			value = `"${keybindingEntry.keybindingItem.keybinding.getAriaLabel()}"`;
+		}
+		else {
+			const [firstPart, chordPart] = keybindingEntry;
+			value = `"${firstPart.getAriaLabel()}`;
+			if (chordPart) {
+				value += ` ${chordPart.getAriaLabel()}`;
+			}
+			value += '"';
+		}
 		if (value !== this.searchWidget.getValue()) {
 			this.searchWidget.setValue(value);
 		}
 		return TPromise.as(null);
 	}
 
+	showOverlayConflicts(keybinding: [ResolvedKeybinding, ResolvedKeybinding]): void {
```

Comment:
- This method should be private
- It takes the `key` as string and call the `fetch` on the model by wrapping the key in quotes.
This method will become a 2 liner like
```typescript
const keybindingsEntries: IKeybindingItemEntry[] = this.keybindingsEditorModel.fetch(`"${keybinding}"`, this.sortByPrecedence.checked);
this.defineKeybindingWidget.printConflicts(keybindingsEntries.length);
```

Reply:
@sandy081 I attempted to make these changes as recommended, but I came across an issue. It seems that using `keybindingsEditorModel.fetch` on certain keybindings such as `"Y"` or `"S D"` will return any items that include these characters on any of their fields, and not necessarily their keybinding field. As you can see in the below screenshot, this behavior is what currently occurs when clicking the "show similar keybindings" button on a particular keybinding:

![image](https://user-images.githubusercontent.com/6752598/39952672-316acaf4-556a-11e8-802c-946ccd44e15a.png)

If this is the intended behavior of that method, that is fine, but it will not work for this purpose, as the length of the list returned  by `keybindingsEditorModel.fetch` will often be much greater than the number of matching keybindings.

If I may make a recommendation, it would be to change the behavior of that method to concatenate a "+" after any characters in the keybinding, as doing so seems to return the results that we're looking for. As a side effect, this would also make the "show similar keybindings" button produce the same behavior, which makes more sense to me than searching through all the fields of the keybinding items for the characters in the keybinding. Please see the below screenshots for my proposed behavior:

![image](https://user-images.githubusercontent.com/6752598/39952727-6c533682-556b-11e8-9e22-c2eb29c89f88.png)

![image](https://user-images.githubusercontent.com/6752598/39952758-afc1ba9c-556b-11e8-95d2-e9602e4d6404.png)

Reply:
@brandonrninefive Agreed to fix the behaviour of the method to match only keybindings when the filter string is quoted.